### PR TITLE
supporting twoside with proc class

### PR DESCRIPTION
**Pull request by FrankMittelbach**
# Internal housekeeping

## Status of pull request

- Ready to merge

## Checklist of required changes before merge will be approved
- [ ] Test file(s) added  --- needed?
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [x] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

### CODE REVIEW

**Review by car222222 — CHANGES_REQUESTED** [review]
In the changes entry:

provided ==>  providing

File: base/changes.txt [hunk 1]
```diff
@@ -6,6 +6,11 @@ completeness or accuracy and it contains some references to files that
 are not part of the distribution.
 ================================================================================
 
+2021-12-09  Frank Mittelbach  <Frank.Mittelbach@latex-project.org>
+
+	* proc.dtx (subsection{The page style}):
+	Support twoside by provided an appropriate footer line (gh/704).
```
Comment (davidcarlisle) [inline base/changes.txt:12]: providing

File: base/proc.dtx [hunk 2]
```diff
@@ -340,7 +340,17 @@
 \let\@evenhead\@empty
 \def\@oddfoot{\normalfont\rightmark \hfil
               \pagename{} \thepage}
-\def\@evenfoot{\@oddfoot}
+%    \end{macrocode}
+%    If the \texttt{twoside} option is passed to the classe we provide
```
Comment (davidcarlisle) [inline base/proc.dtx:344]: class not classe

File: base/proc.dtx [hunk 3]
```diff
@@ -340,7 +340,17 @@
...
+%    a different footer that shows the \cs{leftmark} if that is set.
+% \changes{v1.3m}{2021/12/09}{Support twoside (gh/704)}
+%    \begin{macrocode}
+\if@twoside
+  \def\@evenfoot{\normalfont\pagename{} \thepage
```
Comment (davidcarlisle) [inline base/proc.dtx:349]: the OR has done \reset@font so \normalfont shouldn't really be needed (but proc has apparently always used \normalfont in the foot so perhaps do same here. If its removed same comment applies to the existing setting of \@oddfoot  at line 341 above)

**Review by davidcarlisle — APPROVED** [review]

File: base/proc.dtx (hunk 3 above)
Comment (FrankMittelbach) [reply base/proc.dtx]: might as well remove it

**Pull request merged**